Bonsai/Bonsai.csproj

```diff
@@ -40,45 +38,84 @@
+    <PackageReference Include="System.Memory" Version="4.5.5" GeneratePathProperty="true" PrivateAssets="all" />
+    <PackageReference Include="System.Buffers" Version="4.5.1" GeneratePathProperty="true" PrivateAssets="all" />
+    <PackageReference Include="System.Numerics.Vectors" Version="4.5.0" GeneratePathProperty="true" PrivateAssets="all" />
+    <PackageReference Include="System.Runtime.CompilerServices.Unsafe" Version="6.0.0" GeneratePathProperty="true" PrivateAssets="all" />
+    <EmbeddedResource Include="$(PkgSystem_Resources_Extensions)/lib/net462/System.Resources.Extensions.dll" />
+    <EmbeddedResource Include="$(PkgSystem_Memory)/lib/net461/System.Memory.dll" />
+    <EmbeddedResource Include="$(PkgSystem_Buffers)/lib/net461/System.Buffers.dll" />
+    <EmbeddedResource Include="$(PkgSystem_Numerics_Vectors)/lib/net46/System.Numerics.Vectors.dll" />
+    <EmbeddedResource Include="$(PkgSystem_Runtime_CompilerServices_Unsafe)/lib/net461/System.Runtime.CompilerServices.Unsafe.dll" />
+  </ItemGroup>
+
+  <Target Name="NuGetConfig" AfterTargets="Build">
+    <WriteLinesToFile File="$(TargetDir)NuGet.config"
+                      Lines="$([System.IO.File]::ReadAllText('Properties\NuGet.config').Replace('%PackageDir%','$(PackageOutputPath)'))"
+                      Overwrite="true"
+                      WriteOnlyWhenDifferent="true" />
+  </Target>
+
+  <!-- ================================================================================================================
+    IL Repack Infrastructure
+  ==================================================================================================================-->
+  <Target Name="ConfigureRepack">
+    <PropertyGroup>
+      <RepackedOutputPath>$(BaseOutputPath)$(ArtifactsPivots)-repacked\</RepackedOutputPath>
+      <RepackedOutputFilePath>$(RepackedOutputPath)$(TargetFileName)</RepackedOutputFilePath>
+      <RepackedOutputSymbolFilePath>$([System.IO.Path]::ChangeExtension('$(RepackedOutputFilePath)', '.pdb'))</RepackedOutputSymbolFilePath>
+    </PropertyGroup>
+  </Target>
+
+  <Target Name="Repack" DependsOnTargets="ConfigureRepack">
+    <Error Condition="!$([System.OperatingSystem]::IsWindows())" Text="Repacking is only supported on Windows." />
+    <Error Condition="!Exists('$(OutputPath)Bonsai.exe')" Text="Project must be built before repacking. '$(OutputPath)Bonsai.exe' not found." />
+    <Error Condition="'$(RepackedOutputPath)' == '' or '$(RepackedOutputFilePath)' == ''" Text="Repacked output path not configured." />
+
+    <MakeDir Directories="$(RepackedOutputPath)" />
+
     <PropertyGroup>
-      <WorkingDirectory>$(MSBuildThisFileDirectory)bin\$(Configuration)\$(TargetFramework)</WorkingDirectory>
+      <_IlRepackCommand>
+"$(ILRepack)"
+/verbose
+/out:"$(RepackedOutputFilePath)"
+/internalize:"$(MSBuildThisFileDirectory)InternalizeExcludePatterns"
```

Comment:
I guess this has to come from a file possibly, so no way to just pass the pattern directly into the command line?

Reply:
Unfortunately not. I even dug into the ILRepack source code to double check that I wasn't just misunderstanding something. IIRC the old ILRepack build task we were using was emitting a temporary file to disk for this purpose.